client/trino-client/src/main/java/io/trino/client/ProtocolHeaders.java

```diff
@@ -0,0 +1,203 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.client;
+
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+public final class ProtocolHeaders
+{
+    public static final ProtocolHeaders TRINO_HEADERS = new ProtocolHeaders("Trino");
```

Comment:
The value should be used in client code and tests and not used directly on the server side.
Would it be possible to move the value so that this enforcement is applied at compilation time?